build.sbt

```
@@ -410,6 +412,9 @@ lazy val metals = project
       "org.scala-lang.modules" %% "scala-java8-compat" % "0.9.1",
       "org.scalameta" % "mdoc-interfaces" % V.mdoc,
       "org.scalameta" %% "scalafmt-dynamic" % V.scalafmt,
+      "ch.epfl.scala" % "scalafix-interfaces" % V.scalafix,
+      "ch.epfl.scala" % "scalafix-cli_2.12.12" % V.scalafix,
+      "ch.epfl.scala" % "scalafix-core_2.12" % V.scalafix,
```

is there a specific reason to manually specify the Scala version rather than using `%%`?

Potentially this could be:

```suggestion
      "ch.epfl.scala" %% "scalafix-cli" % V.scalafix cross CrossVersion.full,
      "ch.epfl.scala" %% "scalafix-core" % V.scalafix,
```

but I might be missing some context

---

Thanks for noticing this. Those libs were added only for debugging purposes and should be removed. Thanks.